Dockerfile

```diff
@@ -14,37 +14,13 @@ ENV F90 mpif90
 RUN rm -rf proteus && \
     git clone https://github.com/erdc/proteus && \
     cd proteus && \
-    git checkout master && \
+    git checkout docker_no_opt && \
```

Comment:
I know dockerhub might still not compile, but dockerhub should be tracking when `docker_no_opt` right?